bundles/admin/admin-layereditor/view/AdminLayerForm/PermissionsTabPane.jsx

```diff
@@ -1,7 +1,76 @@
 import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { PermissionRow } from './PermissionTabPane/PermissionRow';
+import { List, ListItem, Checkbox } from 'oskari-ui';
+import { withLocale } from 'oskari-ui/util';
 
-export const PermissionsTabPane = (props) => {
+const StyledListItem = styled(ListItem)`
+    &:first-child > div {
+        font-weight: bold;
+    }
+    &:not(:first-child) {
```

Comment:
TRy using even/odd from css instead of additional prop:
```
tr:nth-child(even) {background: #ffffff}
tr:nth-child(odd) {background: #f3f3f3}
```

Reply:
Fixed as suggested